docs/integrations/delta.md

```diff
@@ -15,11 +15,26 @@ has_children: false
 lakeFS is format-agnostic, so you can save data in Delta format within a lakeFS repository to reap the benefits of both technologies. Specifically:
 
 1. ACID operations can now span across many Delta tables.
-1. [CI/CD hooks](../use_cases/cicd_for_data.md#using-hooks-as-data-quality-gates) can validate Delta table contents, schema, or even referential integrity.
-1. lakeFS supports zero-copy branching for quick experimentation with full isolation.
+2. [CI/CD hooks](../use_cases/cicd_for_data.md#using-hooks-as-data-quality-gates) can validate Delta table contents, schema, or even referential integrity.
+3. lakeFS supports zero-copy branching for quick experimentation with full isolation.
 
 {% include toc.html %}
 
+## Viewing Delta table changes in lakeFS
+
+Using lakeFS, you can compare different versions of Delta tables, view the table operations done since the tables diverged and their details.
+
+
+For example, comparing branches `dev` and `main`, we can see that _movies_ table has changed on `dev` since the branches diverged. 
```

Comment:
```suggestion
For example, comparing branches `dev` and `main`, we can see that the _movies_ table has changed on `dev` since the branches diverged. 
```

Reply:
done